meter/meter_average.go

```diff
@@ -14,8 +14,9 @@ func NewMovingAverageFromConfig(other map[string]interface{}) (api.Meter, error)
 	cc := struct {
 		Decay float64
 		Meter struct {
-			Type  string
-			Other map[string]interface{} `mapstructure:",remain"`
+			Capacity *float64 `mapstructure:"capacity"`
```

Comment:
It's required for technical reasons. It will call a function that requires the value. 
And as I understand, the original meter will not be available at site level, so we need this value for SoC calculation, or am I mistaken here?

Reply:
Why would you use meter_average on batteries?

Reply:
Probably nobody, but why should I implement it in a way that it didn't work if someone did?